indexeddb-examples/idbcursor/style/style.css

```
@@ -0,0 +1,40 @@
+/* || basic set up + sizing for containers */
+
+html, body {
+  margin: 0;
+  font-family: sans-serif;
+  font-size: 10px;
```

While we are here, let’s change this to:

```suggestion
  font: 100%/1.5 sans-serif;
```